db/compaction/compaction_job.cc

```diff
@@ -1797,6 +1797,10 @@ Status CompactionJob::FinishCompactionOutputFile(
   TableProperties tp;
   if (s.ok()) {
     tp = sub_compact->builder->GetTableProperties();
```

Comment:
Please rebase to the latest to resolve conflict. After that, you'd see  `tp = outputs.GetTableProperties();`

Population of meta's field with table properties can be done within `CompactionOutputs` class like https://github.com/facebook/rocksdb/blob/01f2edd145e318228936b89834d058060ed600c6/db/compaction/compaction_outputs.cc#L47-L51

Reply:
Done!